lib/travis/build/git/netrc.rb

```diff
@@ -4,26 +4,30 @@ class Git
       class Netrc < Struct.new(:sh, :data)
         def apply
           sh.fold 'git.netrc' do
```

Comment:
while not directly related to this PR, I don't think we need to use a fold anymore.